gnovm/pkg/gnolang/gno_test.go

```diff
@@ -17,6 +17,92 @@ import (
 	"github.com/jaekwon/testify/require"
 )
 
+func TestRunInvalidLabels(t *testing.T) {
+	tests := []struct {
+		code   string
+		output string
+	}{
+		{
+			code: `
+		package test
+		func main(){}
+		func invalidLabel() {
+			FirstLoop:
+				for i := 0; i < 10; i++ {
+				}
+				for i := 0; i < 10; i++ {
+					break FirstLoop
+				}
+		}
+`,
+			output: `cannot find branch label "FirstLoop"`,
+		},
```

Comment:
For next time, please place tests like these in `gnovm/tests/files`. Goes for all tests which simply run some code and try to test the output or any panic, like these ones

If you're testing specific functions or doing unit tests, then they should go in `gnolang`, but if they are tests which just run some code against the GnoVM then they should go there